src/sass/layout/_header.scss

```diff
@@ -113,7 +117,9 @@
   &:hover,
   &:focus {
     color: $accent-color;
-    fill: $accent-color;
+    -webkit-transition: color $main-duration $main-timing-function;
```

Comment:
Natasha, adding hover and focus to nav has already been done by Vlad Rudenko - see top card in Done list.
Please remove these changes from your commit.